src/xrpld/app/tx/detail/Escrow.cpp

```diff
@@ -533,27 +475,8 @@ EscrowFinish::doApply()
     AccountID const destID = (*slep)[sfDestination];
     auto const sled = ctx_.view().peek(keylet::account(destID));
 
-    if (ctx_.tx.isFieldPresent(sfCredentialIDs))
-    {
-        if (Credentials::removeExpired(view(), ctx_.tx, j_))
-            return tecEXPIRED;
-    }
-    else if (ctx_.view().rules().enabled(featureDepositAuth))
```

Comment:
Why is this check removed?

Reply:
You said that we don't need these checks as they are always enabled

Reply:
My original comments regarding the amendments not required applied to the case where you were adding an amendment guard for `featureCredentials` and you were also adding the guards for two other amendments. In this case the two other guards in fact are not needed because those amendments are enabled. This applies to a newly added code. The comment above applies to removing currently existing guards. While it's true that these amendments are enabled, I don't think we should just remove them when some older code is touched. If we want to remove them, we should have another PR, which removes those guards from everywhere.

Reply:
fixed